server/data/licenceApiClient.ts

```diff
@@ -1,6 +1,7 @@
 import RestClient from './restClient'
-import type { LicenceApiTestData } from './licenceApiClientTypes'
+import type { CreateLicenceResponse, LicenceApiTestData } from './licenceApiClientTypes'
 import config, { ApiConfig } from '../config'
+import { CreateLicenceRequest } from './licenceApiClientTypes'
 
```

Comment:
Can this be combined into the other line above as a 3rd type to import?

Reply:
done